manifest.json

```diff
@@ -1,22 +1,22 @@
 {
-  "serial": 2022102401,
-  "hash": "ujOo9Q4kcpVOdLT5mGImldKlDvdcFvCWs+HVntx+F6w=",
+  "serial": 2022102601,
+  "hash": "YEmTLC7Y/zdhAm+ODRBbdD/D/Gnr+mq46GZVqYZwt9A=",
   "signatures": [
     {
       "address": "13YqCei6dP2ibq2DAy81NmU8FYwwzo5HLKXFA3yM3Rvq9WxDfKF",
       "signature": ""
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
-      "signature": "q/tWU5CpgXTiq9PGvzM1d2ubxN3T9/Dtja3hqI78/HYtf/+s0KgZhbwituwn0uIAfx/rXjvKfejSLPvilbvyAg=="
+      "signature": ""
```

Comment:
```suggestion
      "signature": "pSXzXm7wQBFhjZau+zc/u6baqLnsyKDtF4v/kBzi3+4gZzBuSvmvQU50eb1ncXxhQj5tjd7GmeZeeL/3o7KqCg=="
```

Reply:
@6CornerDefender Can you add #8216, it's a connected with a #7893 but I didn't manage to add it in time and I already had it ready. Thanks